### PR TITLE
TASK: Adjust logger calls for Flow 6.0

### PR DESCRIPTION
**Pull request by daniellienert**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
 		}
 	],
 	"require": {
-		"neos/media": ">3"
+		"neos/media": ">3",
```
Comment (aertmann) [inline composer.json]: could be adjusted to >4 at least? or will there be a version 5 of the package to make that one compatible with Flow 6 as well? if so we wouldn't need the flow requirement here

**Review by aertmann — COMMENTED** [review]
this is for the upcoming Flow v6 version which is only master still right? do you think it makes sense to merge and release a version 4 of this package already?

**Comment by aertmann** [comment]
btw. README should be adjusted as well

**Comment by daniellienert** [comment]
Fixed version constraint and readme. 

Yes, I am currently testing current master on our projects, needs a lost of 3rd party packages to be fixed.
I didn't encounter anymore issues, so it could bee good to go. Merging it to master and releasing it with the Neos version is also fine.

**Pull request merged**